src/main/java/flipkart/dsp/camel/AsyncKafkaProducerConfiguration.java

```diff
@@ -0,0 +1,166 @@
+package flipkart.dsp.camel;
+
+/**
+ * Created by bhushan.sk on 26/08/15.
+ * Kafka Producer Configuration
+ */
+
+
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriParams;
+
+import java.util.Properties;
+
+@UriParams
+public class AsyncKafkaProducerConfiguration {
+
+    @UriParam(defaultValue = "100")
+    private Integer lingerMs = 100;
+
+    @UriParam
+    private String clientId;
+
+    @UriParam(defaultValue = "all")
+    private String acks;
```

Comment:
Although Kafka producer will consider default value for acks as all. But just to be on safer side as we are declaring all to be defaultValue we should set it default as all on our end also.